src/users/users.controller.ts

```diff
@@ -13,11 +15,18 @@ export class UsersController {
         private usersService: UsersService,
     ) {}
 
+    // GET /users
+    // This action returns a user's profile
+    @Get()
+    async getProfile(@GetUser() user: UserDecoratorType): Promise<SanitizedUser> {
+        return await this.usersService.findOne({ id: user.sub });
+    }
+
```

Comment:
`/users` or other routes like such should be, as to follow the rest of the codebase, reserved to admins. It should be marked as `@AdminOnly()` and should return all users. This would make it REST-compliant

Reply:
How do we call a route to get an authenticated user then? *GET users/profile*?

Reply:
Ah, I found your suggestion in a comment below. I'll change it to *users/my* then.